lib/index.js

```diff
@@ -56,7 +58,31 @@ Vero.prototype.group = request('/users/edit', 'put');
  * @api public
  */
 
-Vero.prototype.track = request('/events/track');
+Vero.prototype.track = function(payload, fn) {
+  if (payload.event().match(/[Uu]nsubscribe/)) {
```

Comment:
Why not do this in the mapper? The pattern we use in other integrations is to incorporate the logic re: how to map this data into the `track` mapper; you would then test `payload.event_name` in the handler to determine which endpoint it should go to.

This will also allow you to combine the `trackEvent` and `unsubscribe` handlers into one, since the only difference is the URL.